### DISCUSSION TITLE
Tap native payment using "go_sell_sdk_flutter" Dart package

### DISCUSSION THREAD

**Discussion by saber93** [inspireui/support#5146] [Request new Native Payment]
I'm using Tap payment gateway because it doesn't request company license for activation, no activation fees and support most currencies, moreover they providing wordpress plugin and Dart package called "go_sell_sdk_flutter";

I'd contacted Tap support and got there php and js files for the wordpress plugin to help for Flutter integration.

Kindly find links below for files and Dart package,

https://drive.google.com/drive/folders/1x_fjTDYqF6uWnZmVndC0YETUW58Bvrc0?usp=sharing

https://pub.dev/packages/go_sell_sdk_flutter

Best Regards;

**Comment by hassanyousufx** [comment]
hey can you share the link again? its expired
